server/src/main/java/org/opensearch/index/store/RemoteSegmentStoreDirectory.java

```diff
@@ -127,7 +120,6 @@ public RemoteSegmentStoreDirectory(
      * @throws IOException if there were any failures in reading the metadata file
      */
     public RemoteSegmentMetadata init() throws IOException {
-        this.commonFilenameSuffix = UUIDs.base64UUID();
```

Comment:
should we use `commonFilenameSuffix` in all segment files instead of `UUIDs.base64UUID()` in   `getNewRemoteSegmentFilename`  ?  From just hygiene POV in a follow up PR if it makes sense .

Reply:
We can use same UUID but why would it be more hygenic?

Reply:
All segments from same node would have same suffix . So while listing the segments for any operational reasons we would know who upload it .

Reply:
resolving as it is out of scope for this PR .